.vib/contour/ginkgo/integration_suite_test.go

```diff
@@ -148,3 +153,26 @@ func TestIntegration(t *testing.T) {
 	CheckRequirements()
 	RunSpecs(t, fmt.Sprintf("%s Integration Tests", APP_NAME))
 }
+
+func getBodyOrDie(address string) (string, error) {
```

Comment:
This function isn't really "OrDie" since it's returning an error instead of crashing. I'd change the name.

```suggestion
func getBody(address string) (string, error) {
```